Load correct shared object name in python.
This fixes following error message shown by wireshark/tshark when wireshark-devel
was not installed:
   libwireshark.so: cannot open shared object file: No such file or directory

diff -up wireshark-1.6.0/epan/wspython/wspy_libws.py.soname wireshark-1.6.0/epan/wspython/wspy_libws.py
--- wireshark-1.6.0/epan/wspython/wspy_libws.py.soname	2011-06-09 14:40:04.562726728 +0200
+++ wireshark-1.6.0/epan/wspython/wspy_libws.py	2011-06-09 14:40:09.775467946 +0200
@@ -35,7 +35,7 @@ def get_libws_libname():
   elif system == "Windows":
     return 'libwireshark.dll'
   else:
-    return 'libwireshark.so'
+    return 'libwireshark.so.1'
 
 def get_libws_handle():
   global __libwireshark
